StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/STPPaymentMethodParams.swift

```diff
@@ -1299,6 +1300,8 @@ extension STPPaymentMethodParams {
             return "Revolut Pay"
         case.twint:
             return "TWINT"
+        case .instantDebits:
+            return "Bank Account"
```

Comment:
We have many things named bank account, not sure if we're worried about confusion or if this *should* look the same as an ACHv2 to the consumer

Reply:
Yea agree; many parts, particularly design, are still going to need iterations/feedback from design

There is no public API for this first iteration (everything is behind the scenes), so Android has been merging PR's and iterating